Api/Services/PostingApiService.cs

```diff
@@ -85,10 +90,41 @@ public async Task<PostingResult> Apply(LeverSettings settings, LeverPostingApply
                 var client = _clientFactory.CreateClient();
                 var response = await client.PostAsync($"{URL}{settings.Site}/{model.PostingId}?key={settings.ApiKey}", model.ToFormData());
 
-                return JsonConvert.DeserializeObject<PostingResult>(await response.Content.ReadAsStringAsync());
+                var json = JsonConvert.DeserializeObject<PostingResult>(await response.Content.ReadAsStringAsync());
+
+                if (json.Ok)
+                {
+                    await _workflowManager.TriggerEventAsync(
+                   nameof(LeverPostingNotificationEvent),
+                   input: new
+                   {
+                       LeverPostingNotificationViewModel = new LeverPostingNotificationViewModel
+                       {
+                           IsSuccessful = true,
+                           LeverPostingApplyViewModel = model
+                       }
+                   },
+                   correlationId: model.PostingId
+               );
+                }
+
+                return json;
             }
             catch (Exception e)
```

Comment:
Would including the exception message within the payload sent to the workflow event be useful?